.deployment.config.json

```diff
@@ -0,0 +1,82 @@
+{
+  "product": "coveo-search-ui",
+  "team_name": "searchui",
+  "general": {
+    "aws_regions": {
+      "sequential": ["us-east-1"]
+    },
+    "environments_order": {
+      "sequential": ["dev", "qa", "prd"]
+    },
+    "team_jenkins": "searchuibuilds",
+    "start_environment_automatically": false,
+    "notifications": {
+      "slack_channels": ["#searchuibuilds"]
+    }
+  },
+  "ordered_phases": [
+    {
+      "id": "cdn",
+      "s3": {
+        "bucket": "{terraform.infra.infra.bucket_binaries}",
```

Comment:
Main change is this line which [used to be hard-coded](https://github.com/coveo/search-ui/pull/1762/files#diff-bc6c654bba961cc4c01acff54d583c04301749e1c8c5cfafcb8c0e486aed9f2eL29) to the prod bucket.

Reply:
I didn't know there are terraform variables we could reference